README.md

```diff
@@ -113,6 +113,15 @@ using S3 for storage (the default is `HadoopFileIO` with `HiveCatalog`).
 "iceberg.catalog.io-impl": "org.apache.iceberg.aws.s3.S3FileIO",
 ```
 
+### Nessie example
+```
+"iceberg.catalog.catalog-impl": "org.apache.iceberg.nessie.NessieCatalog",
+"iceberg.catalog.uri": "http://localhost:19120/api/v1",
```

Comment:
maybe rename `localhost` to `nessie` or something similar

Reply:
Since, it is specific to deployment model, I didn't rename.

At Iceberg side we use `localhost` for the Nessie uri
https://iceberg.apache.org/docs/latest/nessie/